### ISSUE TITLE
Add section about team membership

### ISSUE THREAD

**Issue by Splaktar** [gdg-x/code_of_conduct#4]
What is the process here?

Is a merged PR required?

**Comment by tasomaniac** [comment]
Good point. I think merged PR is important. I would say at least X number of PRs in 6 months or something like that.

**Comment by friedger** [comment]
At note to all in team @gdg-x/core, please join a project team. The core team will be removed.

**Comment by Splaktar** [comment]
Just added a link directly to the team policies: https://github.com/gdg-x/code_of_conduct/blob/master/CONTRIBUTING.md#members

As for on boarding new members, how about something like this?
1. PR submitted, reviewed, and merged.
2. Official nomination made in Google+ Community by member of project team where PR was made.
3. Monthly meeting will review nominations and approve/deny.
4. Approved members will be invited to 
   1. **A GDG-X Project team** (I.e. Frisbee) - this gives Write access to the project's repositories and also grants access to the gdg-x private Gitter channel
   2. G+ **GDG-X Members** community - this adds them to the monthly hangout invites

**Comment by tasomaniac** [comment]
1st part should be a necessity 👍 
4th part definitely makes sense.
I think 2 and 3 is too much to handle. Developer's willingness to contribute should be enough. If someone starts to contribute, I don't think there will be a reason to deny

**Comment by Splaktar** [comment]
@tasomaniac So you want something like the following:
1. PR submitted, reviewed, and merged.
2. Monthly meeting will review new contributors with merged PRs to make sure that
3. They are invited to 
   1. **A GDG-X Project team** (I.e. Frisbee) - this gives Write access to the project's repositories and also grants access to the gdg-x private Gitter channel
   2. G+ **GDG-X Members** community - this adds them to the monthly hangout invites

Is there never a case where after 1 merged PR, the existing members would not want to give a person access to our private forums/channels and write access to a set of repositories?

**Comment by tasomaniac** [comment]
I think 1 merged PR is really small. For my understanding, continuity is very important. We always included people when they started submitting PRs regularly. 

I think we should always discuss, but there shouldn't be strict rule and process. We already have a process for removing inactive people. I think that makes sense. 

And also, we shouldn't have very much private forums. I think it is very good now and it is already very limited. 

This is just my opinion. We should get others opinions as well. 

/cc @ozasadnyy @stefanhoth @renfeng @rac2030 @PareshMayani @nitya @fnk0
